### ISSUE TITLE
[FE] Selection doesn't work for page with /Rotate 270

### ISSUE THREAD

**Issue by nikitakovaliov92** [veraPDF/verapdf-webapp-gui#111]
[testFile.pdf](https://github.com/veraPDF/verapdf-webapp-gui/files/4798242/testFile.pdf)

1. Upload attached PDF
2. Select "WCAG 2.1 All" profile and click to inspect button
3. Expand "Pages have different orientation" and click to list item

**Actual result**: Page is not selected
**Expected result**: Page shall be sleected to see validation report error

**Comment by MarinaSaroka** [comment]
Verified
